src/app/tests/TestClusterInfo.cpp

```diff
@@ -73,90 +81,90 @@ void TestAttributePathIntersect(nlTestSuite * apSuite, void * apContext)
         path1.mEndpointId = 1;
         AttributePathParams path2;
         path2.mEndpointId = 2;
-        NL_TEST_ASSERT(apSuite, !path1.Intersects(path2));
+        EXPECT_TRUE(!path1.Intersects(path2));
     }
 
     {
         AttributePathParams path1;
         path1.mClusterId = 1;
         AttributePathParams path2;
         path2.mClusterId = 2;
-        NL_TEST_ASSERT(apSuite, !path1.Intersects(path2));
+        EXPECT_TRUE(!path1.Intersects(path2));
     }
 
     {
         AttributePathParams path1;
         path1.mAttributeId = 1;
         AttributePathParams path2;
         path2.mAttributeId = 2;
-        NL_TEST_ASSERT(apSuite, !path1.Intersects(path2));
+        EXPECT_TRUE(!path1.Intersects(path2));
     }
 }
 
-void TestAttributePathIncludedSameFieldId(nlTestSuite * apSuite, void * apContext)
+TEST_F(TestPath, TestAttributePathIncludedSameFieldId)
 {
     AttributePathParams clusterInfo1;
     AttributePathParams clusterInfo2;
     AttributePathParams clusterInfo3;
     clusterInfo1.mAttributeId = 1;
     clusterInfo2.mAttributeId = 1;
     clusterInfo3.mAttributeId = 1;
-    NL_TEST_ASSERT(apSuite, clusterInfo1.IsAttributePathSupersetOf(clusterInfo2));
+    EXPECT_TRUE(clusterInfo1.IsAttributePathSupersetOf(clusterInfo2));
     clusterInfo2.mListIndex = 1;
-    NL_TEST_ASSERT(apSuite, clusterInfo1.IsAttributePathSupersetOf(clusterInfo2));
+    EXPECT_TRUE(clusterInfo1.IsAttributePathSupersetOf(clusterInfo2));
     clusterInfo1.mListIndex = 0;
-    NL_TEST_ASSERT(apSuite, !clusterInfo1.IsAttributePathSupersetOf(clusterInfo3));
+    EXPECT_TRUE(!clusterInfo1.IsAttributePathSupersetOf(clusterInfo3));
```

Comment:
These were probably regex, however I also wonder if we incrementally update if changing things to `EXPECT_FALSE` would be even clearer.

Reply:
Sure, I plan to fix that sort of stuff later, along with `ASSERT_TRUE(x == y)` -> `ASSERT_EQ(x, y)` mentioned above.